pysc2/lib/renderer_human.py

```diff
@@ -617,6 +630,27 @@ def draw_units(self, surf):
     """Draw the units and buildings."""
     for u, p in self._visible_units():
       if self._camera.intersects_circle(p, u.radius):
+        screen_loc = surf.world_to_surf.fwd_pt(p).floor()
+        aligned_screen_loc = point.Point(screen_loc.x - (self.scale * u.radius * 1.3),
+                                         screen_loc.y - (self.scale * u.radius * 1.35))
+        if u.shield:
+          surf.draw_arc(colors.blue,
+                        aligned_screen_loc,
+                        u.radius,
+                        self.scale * 2.66,
+                        0,
+                        u.shield / (u.shield_max or 1.0) * math.pi,
```

Comment:
Is "or 1.0" the right thing to do here? This would at least avoid the problem of a divide by 0, but would lead to the arc wrapping around multiple times and being useless. It may be better to simply not draw it if you can't figure out the fraction.